### PR TITLE
FE-938 Update extension to match new branding

### PR DESCRIPTION
**Pull request by fireridlle**
Upload GraphQL schema

### CODE REVIEW

**Comment by ryancharris** [comment]
@fireridlle Would you mind adding a screenshot of the logo in the VS Code sidebar to the PR description? Thanks!

**Comment by ryancharris** [comment]
I don't think we want to commit `.DS_STORE` to git here.

File: media/fauna-logo.svg
Comment (ryancharris) [inline media/fauna-logo.svg:2]: You can remove these lines 👍

File: package.json
Comment (ryancharris) [inline package.json:2]: We should leave this as `faunadb` because this is the namespace we publish to on NPM. Unfortunately, there is another project currently using `fauna`, so we may be stick with this.

**Review by ryancharris — CHANGES_REQUESTED** [review]
Left some questions/comments. Looking good though -- thanks!

**Comment by fireridlle** [comment]
> @fireridlle Would you mind adding a screenshot of the logo in the VS Code sidebar to the PR description? Thanks!
sure, i also update screnshoots for readme

**Comment by fireridlle** [comment]
@ryancharris thank you for the review. pushed fixes for all comments

**Review by erickpintor — COMMENTED** [review]
It looks good to me but, I have no context on this codebase. I'll defer to @rodrimaia and @n400 for approvals.

**Review by parkhomenko — APPROVED** [review]

**Comment by n400** [comment]
The code looks fine to me, but let's sanity check it first, using these steps as a manual QA script: https://docs.fauna.com/fauna/current/integrations/vs_code

And then before we can release this, we need to make sure this DOCS issue is ready for release: 
https://faunadb.atlassian.net/browse/DOCS-1523

**Review by n400 — APPROVED** [review]
I'm approving this, but we should wait until the docs change is ready before we release it

**Pull request merged**
